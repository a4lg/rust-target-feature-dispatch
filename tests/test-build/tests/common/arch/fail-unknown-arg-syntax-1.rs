pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    // ERROR: error("error") is not a supported argument syntax.
    if error("error") {
        (1)
    } else {
        0
    }
};
fn main() {}
