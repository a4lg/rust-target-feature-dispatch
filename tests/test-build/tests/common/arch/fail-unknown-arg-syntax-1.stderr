error: Invalid architecture specifier: error("error")
 --> tests/common/arch/fail-unknown-arg-syntax-1.rs:1:24
  |
1 |   pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
  |  ________________________^
2 | |     // ERROR: error("error") is not a supported argument syntax.
3 | |     if error("error") {
4 | |         (1)
... |
8 | | };
  | |_^
  |
  = note: this error originates in the macro `$crate::target_feature_dispatch` which comes from the expansion of the macro `target_feature_dispatch::target_feature_dispatch` (in Nightly builds, run with -Z macro-backtrace for more info)
